lib/publiccloud/azure.pm

```diff
@@ -261,6 +262,7 @@ sub generate_azure_image_definition {
     return get_var('PUBLIC_CLOUD_AZURE_IMAGE_DEFINITION') if (get_var('PUBLIC_CLOUD_AZURE_IMAGE_DEFINITION'));
 
     my $distri = get_required_var('DISTRI');
+    $distri = 'sl-micro' if (is_sle_micro('>=6.0'));
```

Comment:
`sl-micro` ? isn't  correct value `sle-micro`

Reply:
The product has been renamed to `SUSE Linux Micro`, and this is what os-release's ID shows now.

Reply:
district shouldn't be mixed with product so variable name is pretty misleading and here we should use `version_tools`

Reply:
the point is that distri variable used to generate expected image name and while for SLEM before version 6.0 image name contains `sle-micro` ( which matches with distri value ) , for 6.0 image name contains `sl-micro` .  And while I agree with your statements generally I can not follow what exactly you suggesting to address this ?